datadog_downtime unit tests are failing

### Summary

Since apparently [datadog-api-client 1.3.0](https://pypi.org/project/datadog-api-client/1.3.0/) was released today, the unit tests are failing; see for example https://dev.azure.com/ansible/b24bf3ca-6168-45d7-99e2-bf8029e67c87/_apis/build/builds/22716/logs/652:
```
2021-08-16T10:05:12.1708613Z 01:37 =================================== FAILURES ===================================
2021-08-16T10:05:12.1709576Z 01:37 [31m[1m_____________ TestDatadogDowntime.test_create_downtime_when_no_id ______________[0m
2021-08-16T10:05:12.1710418Z 01:37 [gw0] linux -- Python 3.10.0 /usr/bin/python3.10
2021-08-16T10:05:12.1711232Z 01:37 self = <ansible_collections.community.general.tests.unit.plugins.modules.monitoring.test_datadog_downtime.TestDatadogDowntime testMethod=test_create_downtime_when_no_id>
2021-08-16T10:05:12.1712378Z 01:37 downtimes_api_mock = <MagicMock name='DowntimesApi' id='139909948463456'>
2021-08-16T10:05:12.1712901Z 01:37 
2021-08-16T10:05:12.1713490Z 01:37     @patch("ansible_collections.community.general.plugins.modules.monitoring.datadog.datadog_downtime.DowntimesApi")
2021-08-16T10:05:12.1714265Z 01:37     def test_create_downtime_when_no_id(self, downtimes_api_mock):
2021-08-16T10:05:12.1715038Z 01:37         set_module_args({
2021-08-16T10:05:12.1715507Z 01:37             "monitor_tags": ["foo:bar"],
2021-08-16T10:05:12.1715966Z 01:37             "scope": ["*"],
2021-08-16T10:05:12.1716446Z 01:37             "monitor_id": 12345,
2021-08-16T10:05:12.1716973Z 01:37             "downtime_message": "Message",
2021-08-16T10:05:12.1717367Z 01:37             "start": 1111,
2021-08-16T10:05:12.1717625Z 01:37             "end": 2222,
2021-08-16T10:05:12.1717902Z 01:37             "timezone": "UTC",
2021-08-16T10:05:12.1718182Z 01:37             "rrule": "rrule",
2021-08-16T10:05:12.1718459Z 01:37             "api_key": "an_api_key",
2021-08-16T10:05:12.1718778Z 01:37             "app_key": "an_app_key",
2021-08-16T10:05:12.1719023Z 01:37         })
2021-08-16T10:05:12.1719235Z 01:37     
2021-08-16T10:05:12.1719464Z 01:37         downtime = Downtime()
2021-08-16T10:05:12.1719748Z 01:37         downtime.monitor_tags = ["foo:bar"]
2021-08-16T10:05:12.1720234Z 01:37         downtime.scope = ["*"]
2021-08-16T10:05:12.1720703Z 01:37         downtime.monitor_id = 12345
2021-08-16T10:05:12.1721146Z 01:37         downtime.message = "Message"
2021-08-16T10:05:12.1721442Z 01:37         downtime.start = 1111
2021-08-16T10:05:12.1721701Z 01:37         downtime.end = 2222
2021-08-16T10:05:12.1721950Z 01:37         downtime.timezone = "UTC"
2021-08-16T10:05:12.1722242Z 01:37         downtime.recurrence = DowntimeRecurrence(
2021-08-16T10:05:12.1722545Z 01:37             rrule="rrule"
2021-08-16T10:05:12.1722928Z 01:37         )
2021-08-16T10:05:12.1723289Z 01:37     
2021-08-16T10:05:12.1723641Z 01:37 >       create_downtime_mock = MagicMock(return_value=Downtime(id=12345))
2021-08-16T10:05:12.1723923Z 01:37 
2021-08-16T10:05:12.1724210Z 01:37 tests/unit/plugins/modules/monitoring/test_datadog_downtime.py:64: 
2021-08-16T10:05:12.1724598Z 01:37 _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
2021-08-16T10:05:12.1725676Z 01:37 /usr/lib/python3.10/dist-packages/datadog_api_client/v1/model_utils.py:40: in wrapped_init
2021-08-16T10:05:12.1726225Z 01:37     return fn(_self, *args, **kwargs)
2021-08-16T10:05:12.1726817Z 01:37 /usr/lib/python3.10/dist-packages/datadog_api_client/v1/model/downtime.py:203: in __init__
2021-08-16T10:05:12.1727152Z 01:37     super().__init__(kwargs)
2021-08-16T10:05:12.1727482Z 01:37 _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
2021-08-16T10:05:12.1727947Z 01:37 
2021-08-16T10:05:12.1728564Z 01:37 self = {'id': 12345}, kwargs = {'id': 12345}
2021-08-16T10:05:12.1728866Z 01:37 
2021-08-16T10:05:12.1729108Z 01:37     def __init__(self, kwargs):
2021-08-16T10:05:12.1729616Z 01:37         super().__init__(kwargs)
2021-08-16T10:05:12.1730146Z 01:37         for var_name, var_value in kwargs.items():
2021-08-16T10:05:12.1730648Z 01:37             if (
2021-08-16T10:05:12.1731028Z 01:37                 var_name not in self.attribute_map
2021-08-16T10:05:12.1731379Z 01:37                 and self._configuration is not None
2021-08-16T10:05:12.1731754Z 01:37                 and self._configuration.discard_unknown_keys
2021-08-16T10:05:12.1732084Z 01:37                 and self.additional_properties_type is None
2021-08-16T10:05:12.1732572Z 01:37             ):
2021-08-16T10:05:12.1733021Z 01:37                 # discard variable.
2021-08-16T10:05:12.1733414Z 01:37                 continue
2021-08-16T10:05:12.1733701Z 01:37             setattr(self, var_name, var_value)
2021-08-16T10:05:12.1734092Z 01:37             if var_name in self.read_only_vars:
2021-08-16T10:05:12.1734918Z 01:37 >               raise ApiAttributeError(f"`{var_name}` is a read-only attribute.")
2021-08-16T10:05:12.1736052Z 01:37 E               datadog_api_client.v1.exceptions.ApiAttributeError: `id` is a read-only attribute.
2021-08-16T10:05:12.1736751Z 01:37 
2021-08-16T10:05:12.1737326Z 01:37 /usr/lib/python3.10/dist-packages/datadog_api_client/v1/model_utils.py:568: ApiAttributeError
2021-08-16T10:05:12.1737993Z 01:37 [31m[1m________ TestDatadogDowntime.test_create_downtime_when_id_and_disabled _________[0m
2021-08-16T10:05:12.1739047Z 01:37 [gw1] linux -- Python 3.10.0 /usr/bin/python3.10
2021-08-16T10:05:12.1739682Z 01:37 self = <ansible_collections.community.general.tests.unit.plugins.modules.monitoring.test_datadog_downtime.TestDatadogDowntime testMethod=test_create_downtime_when_id_and_disabled>
2021-08-16T10:05:12.1740556Z 01:37 downtimes_api_mock = <MagicMock name='DowntimesApi' id='140702365865760'>
2021-08-16T10:05:12.1741059Z 01:37 
2021-08-16T10:05:12.1741531Z 01:37     @patch("ansible_collections.community.general.plugins.modules.monitoring.datadog.datadog_downtime.DowntimesApi")
2021-08-16T10:05:12.1742268Z 01:37     def test_create_downtime_when_id_and_disabled(self, downtimes_api_mock):
2021-08-16T10:05:12.1742624Z 01:37         set_module_args({
2021-08-16T10:05:12.1743009Z 01:37             "id": 1212,
2021-08-16T10:05:12.1743538Z 01:37             "monitor_tags": ["foo:bar"],
2021-08-16T10:05:12.1743971Z 01:37             "scope": ["*"],
2021-08-16T10:05:12.1744271Z 01:37             "monitor_id": 12345,
2021-08-16T10:05:12.1744635Z 01:37             "downtime_message": "Message",
2021-08-16T10:05:12.1744925Z 01:37             "start": 1111,
2021-08-16T10:05:12.1745194Z 01:37             "end": 2222,
2021-08-16T10:05:12.1745529Z 01:37             "timezone": "UTC",
2021-08-16T10:05:12.1745955Z 01:37             "rrule": "rrule",
2021-08-16T10:05:12.1746454Z 01:37             "api_key": "an_api_key",
2021-08-16T10:05:12.1746794Z 01:37             "app_key": "an_app_key",
2021-08-16T10:05:12.1747108Z 01:37         })
2021-08-16T10:05:12.1747319Z 01:37     
2021-08-16T10:05:12.1747561Z 01:37         downtime = Downtime()
2021-08-16T10:05:12.1748026Z 01:37         downtime.monitor_tags = ["foo:bar"]
2021-08-16T10:05:12.1748548Z 01:37         downtime.scope = ["*"]
2021-08-16T10:05:12.1749008Z 01:37         downtime.monitor_id = 12345
2021-08-16T10:05:12.1749427Z 01:37         downtime.message = "Message"
2021-08-16T10:05:12.1749797Z 01:37         downtime.start = 1111
2021-08-16T10:05:12.1750252Z 01:37         downtime.end = 2222
2021-08-16T10:05:12.1750629Z 01:37         downtime.timezone = "UTC"
2021-08-16T10:05:12.1750987Z 01:37         downtime.recurrence = DowntimeRecurrence(
2021-08-16T10:05:12.1751279Z 01:37             rrule="rrule"
2021-08-16T10:05:12.1751510Z 01:37         )
2021-08-16T10:05:12.1751862Z 01:37     
2021-08-16T10:05:12.1752352Z 01:37 >       create_downtime_mock = MagicMock(return_value=Downtime(id=12345))
2021-08-16T10:05:12.1752797Z 01:37 
2021-08-16T10:05:12.1753095Z 01:37 tests/unit/plugins/modules/monitoring/test_datadog_downtime.py:100: 
2021-08-16T10:05:12.1753658Z 01:37 _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
2021-08-16T10:05:12.1754603Z 01:37 /usr/lib/python3.10/dist-packages/datadog_api_client/v1/model_utils.py:40: in wrapped_init
2021-08-16T10:05:12.1755249Z 01:37     return fn(_self, *args, **kwargs)
2021-08-16T10:05:12.1756015Z 01:37 /usr/lib/python3.10/dist-packages/datadog_api_client/v1/model/downtime.py:203: in __init__
2021-08-16T10:05:12.1756404Z 01:37     super().__init__(kwargs)
2021-08-16T10:05:12.1756796Z 01:37 _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
2021-08-16T10:05:12.1757286Z 01:37 
2021-08-16T10:05:12.1757919Z 01:37 self = {'id': 12345}, kwargs = {'id': 12345}
2021-08-16T10:05:12.1758380Z 01:37 
2021-08-16T10:05:12.1758803Z 01:37     def __init__(self, kwargs):
2021-08-16T10:05:12.1759270Z 01:37         super().__init__(kwargs)
2021-08-16T10:05:12.1759701Z 01:37         for var_name, var_value in kwargs.items():
2021-08-16T10:05:12.1760005Z 01:37             if (
2021-08-16T10:05:12.1760283Z 01:37                 var_name not in self.attribute_map
2021-08-16T10:05:12.1760676Z 01:37                 and self._configuration is not None
2021-08-16T10:05:12.1761230Z 01:37                 and self._configuration.discard_unknown_keys
2021-08-16T10:05:12.1761781Z 01:37                 and self.additional_properties_type is None
2021-08-16T10:05:12.1762338Z 01:37             ):
2021-08-16T10:05:12.1762803Z 01:37                 # discard variable.
2021-08-16T10:05:12.1763174Z 01:37                 continue
2021-08-16T10:05:12.1763491Z 01:37             setattr(self, var_name, var_value)
2021-08-16T10:05:12.1764056Z 01:37             if var_name in self.read_only_vars:
2021-08-16T10:05:12.1765083Z 01:37 >               raise ApiAttributeError(f"`{var_name}` is a read-only attribute.")
2021-08-16T10:05:12.1766181Z 01:37 E               datadog_api_client.v1.exceptions.ApiAttributeError: `id` is a read-only attribute.
2021-08-16T10:05:12.1766791Z 01:37 
2021-08-16T10:05:12.1767654Z 01:37 /usr/lib/python3.10/dist-packages/datadog_api_client/v1/model_utils.py:568: ApiAttributeError
2021-08-16T10:05:12.1768744Z 01:37 [31m[1m___________________ TestDatadogDowntime.test_delete_downtime ___________________[0m
2021-08-16T10:05:12.1769657Z 01:37 [gw1] linux -- Python 3.10.0 /usr/bin/python3.10
2021-08-16T10:05:12.1770424Z 01:37 self = <ansible_collections.community.general.tests.unit.plugins.modules.monitoring.test_datadog_downtime.TestDatadogDowntime testMethod=test_delete_downtime>
2021-08-16T10:05:12.1771585Z 01:37 downtimes_api_mock = <MagicMock name='DowntimesApi' id='140702365871376'>
2021-08-16T10:05:12.1772097Z 01:37 
2021-08-16T10:05:12.1772662Z 01:37     @patch("ansible_collections.community.general.plugins.modules.monitoring.datadog.datadog_downtime.DowntimesApi")
2021-08-16T10:05:12.1773361Z 01:37     def test_delete_downtime(self, downtimes_api_mock):
2021-08-16T10:05:12.1773924Z 01:37         set_module_args({
2021-08-16T10:05:12.1774399Z 01:37             "id": 1212,
2021-08-16T10:05:12.1774875Z 01:37             "state": "absent",
2021-08-16T10:05:12.1775389Z 01:37             "api_key": "an_api_key",
2021-08-16T10:05:12.1775831Z 01:37             "app_key": "an_app_key",
2021-08-16T10:05:12.1776081Z 01:37         })
2021-08-16T10:05:12.1776294Z 01:37     
2021-08-16T10:05:12.1776542Z 01:37         cancel_downtime_mock = MagicMock()
2021-08-16T10:05:12.1776869Z 01:37 >       get_downtime_mock = MagicMock(return_value=Downtime(id=1212))
2021-08-16T10:05:12.1777331Z 01:37 
2021-08-16T10:05:12.1777822Z 01:37 tests/unit/plugins/modules/monitoring/test_datadog_downtime.py:216: 
2021-08-16T10:05:12.1778443Z 01:37 _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
2021-08-16T10:05:12.1779242Z 01:37 /usr/lib/python3.10/dist-packages/datadog_api_client/v1/model_utils.py:40: in wrapped_init
2021-08-16T10:05:12.1779610Z 01:37     return fn(_self, *args, **kwargs)
2021-08-16T10:05:12.1780140Z 01:37 /usr/lib/python3.10/dist-packages/datadog_api_client/v1/model/downtime.py:203: in __init__
2021-08-16T10:05:12.1780612Z 01:37     super().__init__(kwargs)
2021-08-16T10:05:12.1780938Z 01:37 _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
2021-08-16T10:05:12.1781395Z 01:37 
2021-08-16T10:05:12.1782947Z 01:37 self = {'id': 1212}, kwargs = {'id': 1212}
2021-08-16T10:05:12.1783448Z 01:37 
2021-08-16T10:05:12.1783840Z 01:37     def __init__(self, kwargs):
2021-08-16T10:05:12.1784230Z 01:37         super().__init__(kwargs)
2021-08-16T10:05:12.1784621Z 01:37         for var_name, var_value in kwargs.items():
2021-08-16T10:05:12.1784907Z 01:37             if (
2021-08-16T10:05:12.1785185Z 01:37                 var_name not in self.attribute_map
2021-08-16T10:05:12.1785612Z 01:37                 and self._configuration is not None
2021-08-16T10:05:12.1786134Z 01:37                 and self._configuration.discard_unknown_keys
2021-08-16T10:05:12.1786534Z 01:37                 and self.additional_properties_type is None
2021-08-16T10:05:12.1786834Z 01:37             ):
2021-08-16T10:05:12.1787081Z 01:37                 # discard variable.
2021-08-16T10:05:12.1787341Z 01:37                 continue
2021-08-16T10:05:12.1787729Z 01:37             setattr(self, var_name, var_value)
2021-08-16T10:05:12.1788278Z 01:37             if var_name in self.read_only_vars:
2021-08-16T10:05:12.1789443Z 01:37 >               raise ApiAttributeError(f"`{var_name}` is a read-only attribute.")
2021-08-16T10:05:12.1790592Z 01:37 E               datadog_api_client.v1.exceptions.ApiAttributeError: `id` is a read-only attribute.
2021-08-16T10:05:12.1791173Z 01:37 
2021-08-16T10:05:12.1792044Z 01:37 /usr/lib/python3.10/dist-packages/datadog_api_client/v1/model_utils.py:568: ApiAttributeError
2021-08-16T10:05:12.1793125Z 01:37 [31m[1m__________ TestDatadogDowntime.test_update_downtime_when_not_disabled __________[0m
2021-08-16T10:05:12.1793801Z 01:37 [gw1] linux -- Python 3.10.0 /usr/bin/python3.10
2021-08-16T10:05:12.1794273Z 01:37 self = <ansible_collections.community.general.tests.unit.plugins.modules.monitoring.test_datadog_downtime.TestDatadogDowntime testMethod=test_update_downtime_when_not_disabled>
2021-08-16T10:05:12.1795204Z 01:37 downtimes_api_mock = <MagicMock name='DowntimesApi' id='140702365872192'>
2021-08-16T10:05:12.1795690Z 01:37 
2021-08-16T10:05:12.1796258Z 01:37     @patch("ansible_collections.community.general.plugins.modules.monitoring.datadog.datadog_downtime.DowntimesApi")
2021-08-16T10:05:12.1797066Z 01:37     def test_update_downtime_when_not_disabled(self, downtimes_api_mock):
2021-08-16T10:05:12.1797685Z 01:37         set_module_args({
2021-08-16T10:05:12.1798118Z 01:37             "id": 1212,
2021-08-16T10:05:12.1798608Z 01:37             "monitor_tags": ["foo:bar"],
2021-08-16T10:05:12.1798984Z 01:37             "scope": ["*"],
2021-08-16T10:05:12.1799274Z 01:37             "monitor_id": 12345,
2021-08-16T10:05:12.1799565Z 01:37             "downtime_message": "Message",
2021-08-16T10:05:12.1799860Z 01:37             "start": 1111,
2021-08-16T10:05:12.1800261Z 01:37             "end": 2222,
2021-08-16T10:05:12.1800625Z 01:37             "timezone": "UTC",
2021-08-16T10:05:12.1800910Z 01:37             "rrule": "rrule",
2021-08-16T10:05:12.1801198Z 01:37             "api_key": "an_api_key",
2021-08-16T10:05:12.1801544Z 01:37             "app_key": "an_app_key",
2021-08-16T10:05:12.1802007Z 01:37         })
2021-08-16T10:05:12.1802345Z 01:37     
2021-08-16T10:05:12.1802589Z 01:37         downtime = Downtime()
2021-08-16T10:05:12.1802887Z 01:37         downtime.monitor_tags = ["foo:bar"]
2021-08-16T10:05:12.1803173Z 01:37         downtime.scope = ["*"]
2021-08-16T10:05:12.1803540Z 01:37         downtime.monitor_id = 12345
2021-08-16T10:05:12.1804014Z 01:37         downtime.message = "Message"
2021-08-16T10:05:12.1804462Z 01:37         downtime.start = 1111
2021-08-16T10:05:12.1804916Z 01:37         downtime.end = 2222
2021-08-16T10:05:12.1805388Z 01:37         downtime.timezone = "UTC"
2021-08-16T10:05:12.1805888Z 01:37         downtime.recurrence = DowntimeRecurrence(
2021-08-16T10:05:12.1806530Z 01:37             rrule="rrule"
2021-08-16T10:05:12.1806803Z 01:37         )
2021-08-16T10:05:12.1806999Z 01:37     
2021-08-16T10:05:12.1807283Z 01:37 >       update_downtime_mock = MagicMock(return_value=Downtime(id=1212))
2021-08-16T10:05:12.1807567Z 01:37 
2021-08-16T10:05:12.1807886Z 01:37 tests/unit/plugins/modules/monitoring/test_datadog_downtime.py:140: 
2021-08-16T10:05:12.1808455Z 01:37 _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
2021-08-16T10:05:12.1809172Z 01:37 /usr/lib/python3.10/dist-packages/datadog_api_client/v1/model_utils.py:40: in wrapped_init
2021-08-16T10:05:12.1809522Z 01:37     return fn(_self, *args, **kwargs)
2021-08-16T10:05:12.1810433Z 01:37 /usr/lib/python3.10/dist-packages/datadog_api_client/v1/model/downtime.py:203: in __init__
2021-08-16T10:05:12.1811053Z 01:37     super().__init__(kwargs)
2021-08-16T10:05:12.1811425Z 01:37 _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
2021-08-16T10:05:12.1811728Z 01:37 
2021-08-16T10:05:12.1812181Z 01:37 self = {'id': 1212}, kwargs = {'id': 1212}
2021-08-16T10:05:12.1812474Z 01:37 
2021-08-16T10:05:12.1812884Z 01:37     def __init__(self, kwargs):
2021-08-16T10:05:12.1813184Z 01:37         super().__init__(kwargs)
2021-08-16T10:05:12.1813488Z 01:37         for var_name, var_value in kwargs.items():
2021-08-16T10:05:12.1813920Z 01:37             if (
2021-08-16T10:05:12.1814385Z 01:37                 var_name not in self.attribute_map
2021-08-16T10:05:12.1814853Z 01:37                 and self._configuration is not None
2021-08-16T10:05:12.1815196Z 01:37                 and self._configuration.discard_unknown_keys
2021-08-16T10:05:12.1815537Z 01:37                 and self.additional_properties_type is None
2021-08-16T10:05:12.1815812Z 01:37             ):
2021-08-16T10:05:12.1816155Z 01:37                 # discard variable.
2021-08-16T10:05:12.1816611Z 01:37                 continue
2021-08-16T10:05:12.1817092Z 01:37             setattr(self, var_name, var_value)
2021-08-16T10:05:12.1817525Z 01:37             if var_name in self.read_only_vars:
2021-08-16T10:05:12.1818158Z 01:37 >               raise ApiAttributeError(f"`{var_name}` is a read-only attribute.")
2021-08-16T10:05:12.1819175Z 01:37 E               datadog_api_client.v1.exceptions.ApiAttributeError: `id` is a read-only attribute.
2021-08-16T10:05:12.1819787Z 01:37 
2021-08-16T10:05:12.1820390Z 01:37 /usr/lib/python3.10/dist-packages/datadog_api_client/v1/model_utils.py:568: ApiAttributeError
2021-08-16T10:05:12.1821083Z 01:37 [33m=============================== warnings summary ===============================[0m
2021-08-16T10:05:12.1821691Z 01:37 plugins/modules/cloud/misc/proxmox_kvm.py:764
2021-08-16T10:05:12.1822347Z 01:37 plugins/modules/cloud/misc/proxmox_kvm.py:764
2021-08-16T10:05:12.1823320Z 01:37   /root/ansible_collections/community/general/plugins/modules/cloud/misc/proxmox_kvm.py:764: DeprecationWarning: The distutils package is deprecated and slated for removal in Python 3.12. Use setuptools or check PEP 632 for potential alternatives
2021-08-16T10:05:12.1824293Z 01:37     from distutils.version import LooseVersion
2021-08-16T10:05:12.1824714Z 01:37 
```

### Issue Type

Bug Report

### Component Name

datadog_downtime

### Ansible Version

*

### Configuration

*

### OS / Environment

*

### Steps to Reproduce

*

### Expected Results

*

### Actual Results

*

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

--- a/tests/unit/plugins/modules/test_datadog_downtime.py
+++ b/tests/unit/plugins/modules/test_datadog_downtime.py
@@ -7,7 +7,7 @@ from __future__ import (absolute_import, division, print_function)
 
 __metaclass__ = type
 
-from ansible_collections.community.general.plugins.modules.monitoring.datadog import datadog_downtime
+from ansible_collections.community.general.plugins.modules import datadog_downtime
 from ansible_collections.community.general.tests.unit.compat.mock import MagicMock, patch
 from ansible_collections.community.general.tests.unit.plugins.modules.utils import (
     AnsibleExitJson, AnsibleFailJson, ModuleTestCase, set_module_args
@@ -36,7 +36,7 @@ class TestDatadogDowntime(ModuleTestCase):
             set_module_args({})
             self.module.main()
 
-    @patch("ansible_collections.community.general.plugins.modules.monitoring.datadog.datadog_downtime.DowntimesApi")
+    @patch("ansible_collections.community.general.plugins.modules.datadog_downtime.DowntimesApi")
     def test_create_downtime_when_no_id(self, downtimes_api_mock):
         set_module_args({
             "monitor_tags": ["foo:bar"],
@@ -60,10 +60,11 @@ class TestDatadogDowntime(ModuleTestCase):
         downtime.end = 2222
         downtime.timezone = "UTC"
         downtime.recurrence = DowntimeRecurrence(
-            rrule="rrule"
+            rrule="rrule",
+            type="rrule"
         )
 
-        create_downtime_mock = MagicMock(return_value=Downtime(id=12345))
+        create_downtime_mock = MagicMock(return_value=self.__downtime_with_id(12345))
         downtimes_api_mock.return_value = MagicMock(create_downtime=create_downtime_mock)
         with self.assertRaises(AnsibleExitJson) as result:
             self.module.main()
@@ -71,7 +72,7 @@ class TestDatadogDowntime(ModuleTestCase):
         self.assertEqual(result.exception.args[0]['downtime']['id'], 12345)
         create_downtime_mock.assert_called_once_with(downtime)
 
-    @patch("ansible_collections.community.general.plugins.modules.monitoring.datadog.datadog_downtime.DowntimesApi")
+    @patch("ansible_collections.community.general.plugins.modules.datadog_downtime.DowntimesApi")
     def test_create_downtime_when_id_and_disabled(self, downtimes_api_mock):
         set_module_args({
             "id": 1212,
@@ -96,11 +97,16 @@ class TestDatadogDowntime(ModuleTestCase):
         downtime.end = 2222
         downtime.timezone = "UTC"
         downtime.recurrence = DowntimeRecurrence(
-            rrule="rrule"
+            rrule="rrule",
+            type="rrule"
         )
 
-        create_downtime_mock = MagicMock(return_value=Downtime(id=12345))
-        get_downtime_mock = MagicMock(return_value=Downtime(id=1212, disabled=True))
+        disabled_downtime = Downtime()
+        disabled_downtime.disabled = True
+        disabled_downtime.id = 1212
+
+        create_downtime_mock = MagicMock(return_value=self.__downtime_with_id(12345))
+        get_downtime_mock = MagicMock(return_value=disabled_downtime)
         downtimes_api_mock.return_value = MagicMock(
             create_downtime=create_downtime_mock, get_downtime=get_downtime_mock
         )
@@ -111,7 +117,7 @@ class TestDatadogDowntime(ModuleTestCase):
         create_downtime_mock.assert_called_once_with(downtime)
         get_downtime_mock.assert_called_once_with(1212)
 
-    @patch("ansible_collections.community.general.plugins.modules.monitoring.datadog.datadog_downtime.DowntimesApi")
+    @patch("ansible_collections.community.general.plugins.modules.datadog_downtime.DowntimesApi")
     def test_update_downtime_when_not_disabled(self, downtimes_api_mock):
         set_module_args({
             "id": 1212,
@@ -136,11 +142,16 @@ class TestDatadogDowntime(ModuleTestCase):
         downtime.end = 2222
         downtime.timezone = "UTC"
         downtime.recurrence = DowntimeRecurrence(
-            rrule="rrule"
+            rrule="rrule",
+            type="rrule"
         )
 
-        update_downtime_mock = MagicMock(return_value=Downtime(id=1212))
-        get_downtime_mock = MagicMock(return_value=Downtime(id=1212, disabled=False))
+        enabled_downtime = Downtime()
+        enabled_downtime.disabled = False
+        enabled_downtime.id = 1212
+
+        update_downtime_mock = MagicMock(return_value=self.__downtime_with_id(1212))
+        get_downtime_mock = MagicMock(return_value=enabled_downtime)
         downtimes_api_mock.return_value = MagicMock(
             update_downtime=update_downtime_mock, get_downtime=get_downtime_mock
         )
@@ -151,7 +162,7 @@ class TestDatadogDowntime(ModuleTestCase):
         update_downtime_mock.assert_called_once_with(1212, downtime)
         get_downtime_mock.assert_called_once_with(1212)
 
-    @patch("ansible_collections.community.general.plugins.modules.monitoring.datadog.datadog_downtime.DowntimesApi")
+    @patch("ansible_collections.community.general.plugins.modules.datadog_downtime.DowntimesApi")
     def test_update_downtime_no_change(self, downtimes_api_mock):
         set_module_args({
             "id": 1212,
@@ -176,7 +187,8 @@ class TestDatadogDowntime(ModuleTestCase):
         downtime.end = 2222
         downtime.timezone = "UTC"
         downtime.recurrence = DowntimeRecurrence(
-            rrule="rrule"
+            rrule="rrule",
+            type="rrule"
         )
 
         downtime_get = Downtime()
@@ -205,7 +217,7 @@ class TestDatadogDowntime(ModuleTestCase):
         update_downtime_mock.assert_called_once_with(1212, downtime)
         get_downtime_mock.assert_called_once_with(1212)
 
-    @patch("ansible_collections.community.general.plugins.modules.monitoring.datadog.datadog_downtime.DowntimesApi")
+    @patch("ansible_collections.community.general.plugins.modules.datadog_downtime.DowntimesApi")
     def test_delete_downtime(self, downtimes_api_mock):
         set_module_args({
             "id": 1212,
@@ -215,12 +227,16 @@ class TestDatadogDowntime(ModuleTestCase):
         })
 
         cancel_downtime_mock = MagicMock()
-        get_downtime_mock = MagicMock(return_value=Downtime(id=1212))
         downtimes_api_mock.return_value = MagicMock(
-            get_downtime=get_downtime_mock,
+            get_downtime=self.__downtime_with_id,
             cancel_downtime=cancel_downtime_mock
         )
         with self.assertRaises(AnsibleExitJson) as result:
             self.module.main()
         self.assertTrue(result.exception.args[0]['changed'])
         cancel_downtime_mock.assert_called_once_with(1212)
+
+    def __downtime_with_id(self, id):
+        downtime = Downtime()
+        downtime.id = id
+        return downtime

